src/Business Foundation/App/NoSeries/src/Setup/NoSeriesType.Enum.al

```diff
@@ -0,0 +1,17 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Foundation.NoSeries;
+
```

Comment:
Add comment block, with summary e.g.:
```suggestion

/// <summary>
/// The type to which the No. Series belongs. Please extend this enum to add your own type to classify the No. Series being added by your extension.
/// </summary>

```

Reply:
This type is still WIP and is likely to be removed from here. It's currently used to support IT functionality. We will of course add docs if it's kept!